client/lib/happychat/connection.js

```diff
@@ -151,4 +161,4 @@ class Connection extends EventEmitter {
 	}
 }
 
-export default () => new Connection();
+export default ( ) => new Connection( );
```

Comment:
I think we've adopted the convention of not adding these spaces. so the norm is `() => new Connection();`

Reply:
Fixed in 4646789